apps/ledger-live-desktop/src/renderer/screens/accounts/AccountRowItem/index.tsx

```diff
@@ -145,7 +145,13 @@ const AccountRowItem = (props: Props) => {
 
   const scrollTopFocusRef = useRef<HTMLSpanElement>(null);
 
+  const mountedRef = useRef(true);
```

Comment:
This will disable the auto scroll but in DEV every change done on the page will result in an auto scroll to the last account